### ISSUE TITLE
Slow performance

### ISSUE THREAD

**Issue by shukla-yogesh** [microsoft/vscode#109079]
Issue Type: <b>Bug</b>

non response to commands and very high memory and cpu usage.

VS Code version: Code 1.50.1 (d2e414d9e4239a252d1ab117bd7067f125afd80a, 2020-10-13T15:06:15.712Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.85GB (10.89GB free)|
|Process Argv|--crash-reporter-id 57b418f4-0ce1-4e84-be24-48ed7f995e64|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
salesforce-diff|chu|0.0.7
vlocode|cur|0.14.9
vscode-eslint|dba|2.1.13
prettier-vscode|esb|5.7.1
beautify|Hoo|1.5.0
compareit|in4|0.0.2
git-tree-compare|let|1.10.0
vscode-xml|red|0.13.0
salesforce-vscode-slds|sal|1.3.0
salesforcedx-vscode|sal|50.1.0
salesforcedx-vscode-core|sal|50.1.0
salesforcedx-vscode-lightning|sal|50.1.0
salesforcedx-vscode-lwc|sal|49.8.0
sfdx-package-xml-generator|Vig|2.0.1

</details>

**Comment by brianamm** [comment]
I dropped by here today to see if anyone else was seeing performance issues. The last version that performed well for me is 1.48.2, which I've reverted back to. I didn't look at CPU or memory usage before uninstalling 1.50.1 (though, I should go back and do that), so not sure we're looking at the same problem here, but I'll throw in 2 cents in case it's helpful. What I'm seeing is very slow performance scrolling through files (small/250 lines, medium/2500 lines, and large/250,000 lines). "Find" is also very slow, but basically it's the same issue. That is, when I start typing into the "find" box, VSCode starts live scrolling to the next match, which is very pokey. Disabling all extensions made no difference. VSCode versions 1.48.2 and below are smooth and responsive with the same operations on the same files.
